C/kmsg_log.c

```diff
@@ -10,45 +10,45 @@
  #define LOG_LOC "/cache/lible.log"
 #endif
 
-
+/*
+  This function just basically takes the input
+  arg and prints it to /dev/kmsg.
+  Its final printing point can be changed from
+  /dev/kmsg to any other location
+  using the Header LOG_LOC
+  currently if it cant open kmsg it logs it
+  in the /cache/lible.log
+  */
 int klog(char *msg)
 {
 
  char *buf = malloc(256);
  int ret = -1;
 
-  // Write to kmsg if possibe.
```

Comment:
A briefing about what a function does is enough. We dont need to Comment everything , our code should be able to explain whats happening

Comment when you sense reader wont know whats happening